test/functional/basic/gesture-e2e-specs.js

```diff
@@ -1,310 +1,363 @@
+    // TODO: Need a scrollable screen.
+    it.skip(`should swipe the table and the bottom cell's Y position should change accordingly`, async function () {
+      const winEl = await driver.$(`${PREDICATE_SEARCH}:type == 'XCUIElementTypeWindow'`);
+
+      const pickerEl = await driver.$('~Picker View');
+      const loc = await pickerEl.getLocation();
+
+      await driver.execute('mobile: swipe', {element: winEl, direction: 'up'}).should.not.be.rejected;
+      const locMiddle = await pickerEl.getLocation();
+      locMiddle.y.should.be.below(loc.y);
+
+      await driver.execute('mobile: swipe', {element: winEl, direction: 'down'}).should.not.be.rejected;
+      const locFinal = await pickerEl.getLocation();
+      locFinal.y.should.be.above(locMiddle.y);
+    });
+    describe('pinch and zoom', function () {
+      beforeEach(async function () {
+        await driver.execute('mobile: scroll', {direction: 'down'});
+        const el = await driver.$('~Web View');
+        await el.click();
+      });
+
+      // at this point this test relies on watching it happen, nothing is asserted
+      // in automation, this just checks that errors aren't thrown
+      it('should be able to pinch', async function () {
+        // const ctxs = await driver.execute('mobile: getContexts', {waitForWebviewMs: 30000});
+        const ctxs = await driver.getContexts();
+        await driver.switchContext(ctxs[1]);
+
+        await driver.url(APPIUM_IMAGE);
+
+        await driver.switchContext(ctxs[0]);
+
+        async function doZoom () {
+          const el = await driver.$(`${PREDICATE_SEARCH}:type == 'XCUIElementTypeApplication'`);
+          const thumb = driver.action('pointer')
+            .move({ origin: el, x: 100, y: 0 })
+            .down()
+            .pause(100)
+            .move({ origin: el, x: 50, y: 0 })
+            .up();
+          const foreFinger = driver.action('pointer')
+            .move({ origin: el, x: 100, y: 0 })
+            .down()
+            .pause(100)
+            .move({ origin: el, x: 105, y: 0 })
+            .up();
+
+          await driver.actions([thumb, foreFinger]);
+        }
+        await doZoom();
+
+        async function doPinch () {
+          const el = await driver.$(`${PREDICATE_SEARCH}:type == 'XCUIElementTypeApplication'`);
+          const thumb = driver.action('pointer')
+            .move({ origin: el, x: 50, y: 0 })
+            .down()
+            .pause(100)
+            .move({ origin: el, x: 100, y: 0 })
+            .up();
+          const foreFinger = driver.action('pointer')
+            .move({ origin: el, x: 100, y: 0 })
+            .down()
+            .pause(100)
+            .move({ origin: el, x: 50, y: 0 })
+            .up();
+
+          await driver.actions([thumb, foreFinger]);
+        }
+        await doPinch();
+      });
+    });
+    describe('special actions', function () {
+      // TODO: Neither ControlCenterView or ControlCenterView is available on recent iOS.
```

Comment:
I don't have an idea on how to deal with this test 🥺

Reply:
just delete the test completely, it does too many assumptions

Reply:
Thanks! deleted https://github.com/appium/appium-xcuitest-driver/pull/1479/commits/3a47d445a865adccc90009dd5df1d1d8c07fde59